src/components/progress-tracker/progress-tracker.stories.mdx

```diff
@@ -460,3 +265,19 @@ This example shows the automatic calculation of `aria-valuenow` when not provide
 ### ProgressTracker
 
 <StyledSystemProps of={ProgressTracker} noHeader margin />
+
+## Translation keys
+
+The following keys are available to override the translations for this component by passing in a custom locale object
+to the [i18nProvider](https://carbon.sage.com/?path=/story/documentation-i18n--page).
+
+<TranslationKeysTable
+  translationData={[
+    {
+      name: "progressTracker.ofLabel",
```

Comment:
`progressTracker.ofLabel` -> `progressTracker.of`

Reply:
Hi @edleeks87  - first change was a result of failing tests - it has been resolved and branch has been updated. I reached out to Igor Dlugosh regarding failing Cypress tests. Once all tests are passing I will amend the commit message accordingly. Thank you for your feedback :)